src/test/resources/hudson/security/planetexpressExtGroups_withCn.ldif

```diff
@@ -0,0 +1,2506 @@
+dn: ou=people,dc=planetexpress,dc=com
+objectClass: top
+objectClass: organizationalUnit
+description: Planet Express crew
+ou: people
+
+dn: ou=groups,dc=planetexpress,dc=com
+objectclass: organizationalUnit
+objectclass: top
+description: Contains entries which describe groups
+ou: groups
+
+dn: cn=Amy Wong+sn=Kroker,ou=people,dc=planetexpress,dc=com
+objectClass: top
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+cn: Amy Wong
+sn: Kroker
+description: Human
+givenName: Amy
+mail: amy@planetexpress.com
+ou: Intern
+uid: amy
+userPassword:: e3NzaGF9M3UzcUdCSmFMc2tiUEg0OVJrYlFtUk9HTktFb1lOUXZkU2lOZmc9P
+ Q==
+
+dn: cn=Bender Bending Rodríguez,ou=people,dc=planetexpress,dc=com
+objectClass: inetOrgPerson
+objectClass: organizationalPerson
+objectClass: person
+objectClass: top
+cn: Bender Bending Rodríguez
+sn: Bending Rodríguez
+description: Robot
+displayName: Bender
+employeeType: Ship's Robot
+givenName: Bender
+jpegPhoto:: /9j/4AAQSkZJRgABAQEAYABgAAD/4QBgRXhpZgAASUkqAAgAAAACADEBAgAHAAAA
```

Comment:
I don't think you need to pollute the repo with more unused data.

Reply:
done